### ISSUE TITLE
Cloudflare Workers support

### ISSUE THREAD

**Issue by Zerebokep** [bevry/istextorbinary#293]
Any plans to fully support cloudflare workers (or serverless in general)? [this](https://github.com/bevry/istextorbinary/issues/292) is a good first step, but there should also be a version which doesn't rely on fs, e.g. by passing the filename and file data by string/uint8array.

**Comment by balupton** [comment]
Good idea. I recently added cloudflare worker support for bevry/startofweek which is used in bevry/daet for bevry/billing. The support needs to be rolled out to bevry/boundation then any bevry package can add a workerd file with custom support.

My current focus is just keeping my head above water. Is this a request for your employer? Would they be interested in sponsoring me or funding the feature?
